armi/operators/settingsValidation.py

```diff
@@ -224,6 +224,36 @@ def run(self, cs=None):
                 )
             runLog.debug("{} has finished querying.".format(self.__class__.__name__))
 
+        # decide what to do with original settings
+        if correctionsMade:
+            oldFilePath = "{}_old.yaml".format(self.cs.path.split(".yaml")[0])
+            if self._csRelativePathExists(oldFilePath):
+                overwrite_file = prompt(
+                    'INSPECTOR: At least one corrective query has been resolved and an "_old.yaml" settings file {} exists.'.format(
+                        oldFilePath
+                    ),
+                    'Overwrite "_old.yaml" settings file to preserve originally submitted settings?',
+                    "YES_NO",
+                    "NO_DEFAULT",
+                    "CANCEL",
+                )
+                if overwrite_file:
+                    self.csOriginal.writeToYamlFile(oldFilePath)
```

Comment:
please change this just to a rename. loading int memory and then rewritting will suppress default settings and a may delete comments/change formatting. we want the exact original file.

Reply:
Ok I have changed this to copy the file and rename instead of reading into memory.